git_machete/client.py

```diff
@@ -1383,34 +1383,41 @@ def is_excluded_reflog_subject(sha_: str, gs_: str) -> bool:
               "and branch reset events irrelevant for fork point/upstream inference): %s\n" % (", ".join(result) or "<empty>"))
         return result
 
+    def get_remote_name_and_organization_and_repository_name_of_remote(self) -> 'RemoteAndOrganizationAndRepository':
+        return RemoteAndOrganizationAndRepository(remote=self.__git.get_config_attr_or_none(f"machete.github.remote"),
```

Comment:
Okay! if you could provide docs for that config... I think it should be treated similarly as the docs for GitHub authentication, i.e. shared b/w multiple locations (`include`d in .rst)